homeassistant/helpers/selector.py

```diff
@@ -358,6 +358,38 @@ def __call__(self, data: Any) -> str:
         return config
 
 
+class ConstantSelectorConfig(TypedDict, total=False):
+    """Class to represent a constant selector config."""
+
+    value: str | int | bool
+    label: str
+
+
+@SELECTORS.register("constant")
+class ConstantSelector(Selector[ConstantSelectorConfig]):
+    """Constant selector."""
+
+    selector_type = "constant"
+
+    CONFIG_SCHEMA = vol.Schema(
+        {
+            vol.Optional("value"): vol.Any(str, int, bool),
```

Comment:
Why is this optional?

Reply:
A constant without a value -> should we accept `None` then ?

Reply:
I think this should be resolved in the frontend PR (I had the same question about why the value is optional).

Reply:
I changed it to required in frontend 👍

Reply:
OK, I updated this PR accordingly